[quant][graphmode][fx][refactor] Move patterns to separate files

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43984 [quant][graphmode][fx][api] Call fuse in prepare
* #43983 [quant][graphmode][fx] Support inplace option
* #43952 [quant][graphmode][fx] Support dynamic quantization without calibration
* #43892 [quant][graphmode][fx][refactor] Factor out graph extraction code for reuse
* **#43891 [quant][graphmode][fx][refactor] Move patterns to separate files**

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23429759](https://our.internmc.facebook.com/intern/diff/D23429759)

jerryzh168 committed:
[quant][graphmode][fx][refactor] Move patterns to separate files

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

vkuzo reviewed (approved):
lg, thanks for doing this!

jerryzh168 committed:
Update on "[quant][graphmode][fx][refactor] Move patterns to separate files"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23429759](https://our.internmc.facebook.com/intern/diff/D23429759)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx][refactor] Move patterns to separate files"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23429759](https://our.internmc.facebook.com/intern/diff/D23429759)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx][refactor] Move patterns to separate files"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23429759](https://our.internmc.facebook.com/intern/diff/D23429759)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx][refactor] Move patterns to separate files"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23429759](https://our.internmc.facebook.com/intern/diff/D23429759)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx][refactor] Move patterns to separate files"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23429759](https://our.internmc.facebook.com/intern/diff/D23429759)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx][refactor] Move patterns to separate files"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23429759](https://our.internmc.facebook.com/intern/diff/D23429759)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx][refactor] Move patterns to separate files"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23429759](https://our.internmc.facebook.com/intern/diff/D23429759)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx][refactor] Move patterns to separate files"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23429759](https://our.internmc.facebook.com/intern/diff/D23429759)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx][refactor] Move patterns to separate files"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23429759](https://our.internmc.facebook.com/intern/diff/D23429759)

[ghstack-poisoned]
